decomp2gef.py

```diff
@@ -22,11 +22,12 @@
 import functools
 import struct
 import os
-
+import hashlib
```

Comment:
PEP8 import formatting:
<built in python libraries>

<3rd party libraries>

<local libraries>

Please reformat this since `hashlib` is built-in and `sortedcontainers` is 3rd party (requires install)